src/Cell.cpp

```diff
@@ -453,123 +591,21 @@ double Cell::minSurfaceDist(Point* point, double angle,
 
 
 /**
- * @brief Constructor sets the user-specified and unique IDs for this CellBasic.
- * @param id the user-specified optional Cell ID
- * @param name the user-specified optional Cell name
- * @param rings the number of equal volume rings to divide this Cell into
- *        (the default is zero)
- * @param sectors the number of angular sectors to divide this Cell into
- *        (the default is zero)
- */
-CellBasic::CellBasic(int id, const char* name, int rings, int sectors):
-  Cell(id, name) {
-
-  _cell_type = MATERIAL;
-  setNumRings(rings);
-  setNumSectors(sectors);
-}
-
-
-
-/**
- * @brief Return the Material filling the CellBasic.
- * @return the Material's pointer
- */
-Material* CellBasic::getMaterial() {
-  return _material;
-}
-
-
-/**
- * @brief Return the number of rings in the Cell.
- * @return the number of rings
- */
-int CellBasic::getNumRings() {
-  return _num_rings;
-}
-
-
-/**
- * @brief Return the number of sectors in the Cell.
- * @return the number of sectors
- */
-int CellBasic::getNumSectors() {
-  return _num_sectors;
-}
-
-
-/**
- * @brief Returns an empty std::map of Cell IDs and Cell pointers.
- * @return empty std::map of Cell IDs and pointers
- */
-std::map<int, Cell*> CellBasic::getAllCells() {
-  std::map<int, Cell*> cells;
-  return cells;
-}
```

Comment:
I guess you kept the previous functionality.  It just doesn't make much sense to me!